packages/datagateway-dataview/public/res/default.json

```diff
@@ -72,6 +82,9 @@
         "label": "Publications",
         "reference": "Reference"
       }
+    },
+    "isis": {
+      "name":"RB Number"
```

Comment:
I'm not exactly a fan of this "isis" section header - the idea would be that the facility would configure `investigation.name` to be either `RB Number` in ISIS, or `Name` for DLS.

Reply:
Ah OK, the original issue mentioned making RB numbers more apparent or something like that so I thought I needed to change something to say "RB Number" as part of the change, but it being configurable by the user makes sense. Provided we're displaying the `<entity>.name` for relevant tables/views it should be OK then?